MonoGame.Framework/Point3.cs

```diff
@@ -0,0 +1,346 @@
+// MIT License - Copyright (C) The Mono.Xna Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
```

Comment:
Go ahead and change this to the MonoGame copyright since it is new code:
```
// MonoGame - Copyright (C) The MonoGame Team
// This file is subject to the terms and conditions defined in
// file 'LICENSE.txt', which is part of this source code package.
```